### ISSUE TITLE
Error initializing arcadia.packages.clj 

### ISSUE THREAD

**Issue by selfsame** [arcadia-unity/Arcadia#252]
when `Arcadia>Compiler>AOT Compile Internal Namepaces` has been done, the following error happens during flush, and the (UDP) repl doesn't automatically start

```
clojure.lang.Compiler+AssemblyInitializationException: Error initializing arcadia.packages.clj, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> clojure.lang.Compiler+AssemblyInitializationException: Error initializing arcadia.internal.mozroots.clj, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.MissingMethodException: Cannot call instance method named GetMethod on nil
at (wrapper dynamic-method) object.CallSite.Target (System.Runtime.CompilerServices.Closure,System.Runtime.CompilerServices.CallSite,object,object,object) <IL 0x00010, 0x00095>
at (wrapper dynamic-method) object.CallSite.Target (System.Runtime.CompilerServices.Closure,System.Runtime.CompilerServices.CallSite,object,object,object) <IL 0x00142, 0x009f4>
at __Init__$arcadia$internal$mozroots.__interop_GetMethod58714 (object,object,object) <IL 0x00010, 0x00091>
at __Init__$arcadia$internal$mozroots.Initialize () <IL 0x00238, 0x01831>
at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke (object,object[],System.Exception&) <IL 0x0001b, 0x00103>
at System.Reflection.MonoMethod.Invoke (object,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo) [0x000d0] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222

  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x000eb] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:232 
  at System.MonoType.InvokeMember (System.String name, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object target, System.Object[] args, System.Reflection.ParameterModifier[] modifiers, System.Globalization.CultureInfo culture, System.String[] namedParameters) [0x00329] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/MonoType.cs:433 
  at System.Type.InvokeMember (System.String name, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object target, System.Object[] args) [0x00000] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Type.cs:1149 
  at clojure.lang.Compiler.InvokeInitType (System.Reflection.Assembly assy, System.Type initType) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at clojure.lang.Compiler.InvokeInitType (System.Reflection.Assembly assy, System.Type initType) [0x00000] in <filename unknown>:0 
  at clojure.lang.Compiler.InitAssembly (System.Reflection.Assembly assy, System.String relativePath) [0x00000] in <filename unknown>:0 
  at clojure.lang.Compiler.LoadAssembly (System.IO.FileInfo assyInfo, System.String relativePath) [0x00000] in <filename unknown>:0 
  at clojure.lang.RT.load (System.String relativePath, Boolean failIfNotFound) [0x00000] in <filename unknown>:0 
  at clojure.lang.RT.load (System.String relativePath) [0x00000] in <filename unknown>:0 
  at clojure/core$loadfn__38461__38465.invoke () [0x00000] in <filename unknown>:0 
  at clojure/core$load__38468.invokeStatic (ISeq ) [0x00000] in <filename unknown>:0 
  at clojure/core$load__38468.doInvoke (System.Object ) [0x00000] in <filename unknown>:0 
  at clojure.lang.RestFn.invoke (System.Object arg1) [0x00000] in <filename unknown>:0 
  at clojure/core$load_one__38341.invokeStatic (System.Object , System.Object , System.Object ) [0x00000] in <filename unknown>:0 
  at clojure/core$load_one__38341.invoke (System.Object , System.Object , System.Object ) [0x00000] in <filename unknown>:0 
  at clojure/core$load_libfn__38379__38383.invoke () [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x000eb] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:232 
  at System.MonoType.InvokeMember (System.String name, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object target, System.Object[] args, System.Reflection.ParameterModifier[] modifiers, System.Globalization.CultureInfo culture, System.String[] namedParameters) [0x00329] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/MonoType.cs:433 
  at System.Type.InvokeMember (System.String name, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object target, System.Object[] args) [0x00000] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Type.cs:1149 
  at clojure.lang.Compiler.InvokeInitType (System.Reflection.Assembly assy, System.Type initType) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at clojure.lang.Compiler.InvokeInitType (System.Reflection.Assembly assy, System.Type initType) [0x00000] in <filename unknown>:0 
  at clojure.lang.Compiler.InitAssembly (System.Reflection.Assembly assy, System.String relativePath) [0x00000] in <filename unknown>:0 
  at clojure.lang.Compiler.LoadAssembly (System.IO.FileInfo assyInfo, System.String relativePath) [0x00000] in <filename unknown>:0 
  at clojure.lang.RT.load (System.String relativePath, Boolean failIfNotFound) [0x00000] in <filename unknown>:0 
  at clojure.lang.RT.load (System.String relativePath) [0x00000] in <filename unknown>:0 
  at Arcadia.Initialization.LoadPackages () [0x00010] in /home/selfsame/unity/tests/Assets/Arcadia/Editor/Initialization.cs:84 
  at Arcadia.Initialization.Initialize () [0x00015] in /home/selfsame/unity/tests/Assets/Arcadia/Editor/Initialization.cs:66 
  at Arcadia.Initialization..cctor () [0x00001] in /home/selfsame/unity/tests/Assets/Arcadia/Editor/Initialization.cs:15 
UnityEditor.EditorAssemblies:ProcessInitializeOnLoadAttributes()
```

**Comment by nasser** [comment]
Which commit is this on?

**Comment by selfsame** [comment]
481708346539a268a9d5321e09b561626eb789c0 (develop)
might be linux related?

**Comment by timsgardner** [comment]
fixed on cc163ba

It wasn't linux, looks like something annoying with type symbol resolution muckity muck when dynamically loading assemblies, then importing their types
